### PR TITLE
Add grinder rebuild option in the end of test target

### PR DESCRIPTION
**Pull request by renfeiw**
[ci skip]

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
reviewer: @llxia

File: test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl [hunk 1]
```diff
@@ -186,6 +194,20 @@ sub resultReporter {
 
 	print "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n";
 
+	if ($numOfFailed != 0) {
+		my $buildParam =  "";
+		if ($buildList ne '') {
+			$buildParam = "&BUILD_LIST=" . $buildList;
+		}
+		print "To rebuild the failed test in a jenkins job, copy the following link and fill out the <Jenkins URL> and <FAILED test target>:\n";
+		print "<Jenkins URL>/parambuild/?PERSONAL_BUILD=false&JDK_VERSION=$jdkVersion&JDK_IMPL=$jdkImpl$buildParam&TARGET=<FAILED test target>\n\n";
```
Comment (llxia) [inline test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl]: Please remove `PERSONAL_BUILD=false`. This is no longer needed.

Comment (renfeiw) [reply test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl]: removed

File: test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl [hunk 2]
```diff
@@ -30,6 +30,26 @@
...
+	'linux_ppc-64_cmprssptrs'    => 'openjdk_ppc64_aix',
+	'linux_ppc-64_cmprssptrs_le' => 'openjdk_ppc64le_linux',
+	'linux_390-64_cmprssptrs'    => 'openjdk_s390x_linux',
+	'zos_390-64_cmprssptrs'      => 'openjdk_s390x_zos',
+	'osx_x86-64_cmprssptrs'      => 'openjdk_x86-64_mac',
+	'osx_x86-64'                 => 'openjdk_x86-64_mac_xl',
+	'win_x86-64_cmprssptrs'      => 'openjdk_x86-64_windows',
+	'win_x86-64'                 => 'openjdk_x86-64_windows_xl',
+	'win_x86'                    => 'openjdk_x86-32_windows',
+	'osx_sparcv9-64'             => 'openjdk_sparc_solaris',
```
Comment (llxia) [inline test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl]: `osx_sparcv9-64` will be changed soon https://github.com/AdoptOpenJDK/openjdk-tests/pull/1319

File: test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl [hunk 3]
```diff
@@ -30,6 +30,26 @@
...
 my $diagnostic = 'failure';
+my $jdkVersion = "";
+my $jdkImpl = "";
+my $buildList = "";
+my $spec = "";
+my %spec2jenkinsFile = (
+	'linux_x86-64_cmprssptrs'    => 'openjdk_x86-64_linux',
+	'linux_x86-64'               => 'openjdk_x86-64_linux_xl',
+	'linux_arm'                  => 'openjdk_aarch32_linux',
+	'linux_aarch64_cmprssptrs'   => 'openjdk_aarch64_linux',
```
Comment (llxia) [inline test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl]: `linux_aarch64` is missing

**Comment by llxia** [comment]
Instead of hardcode the map, will be easier to leverage `arch` and `os` variables?

**Comment by renfeiw** [comment]
We will use a hardcoded map at this moment. We can remove the map once the Jenkins file name is matched with the spec.

File: test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl (hunk 2 above)
Comment (renfeiw) [reply test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl]: changed based on the new name

**Comment by llxia** [comment]
jenkins test sanity plinux jdk11

**Pull request merged**